.github/workflows/build.yml

```
@@ -203,7 +203,7 @@ jobs:
         fi
 
         # Check that commit contains ':' (and no autosquashable commit using '!')
-        if commit_titles | grep -E -v '^([-.a-z]+[,:] )*[-.a-z]+: [a-z][a-z0-9 _.,+-`/]*$'; then
+        if commit_titles | grep -E -v '^([-.a-z]+[,:] )*[-.a-z]+: [a-z][a-zA-Z0-9 _.,+`/\-]*$'; then
```

```suggestion
        if commit_titles | grep -E -v '^([-.a-z]+[,:] )*[-.a-z]+: [a-z][a-Z0-9 _.,+`/\-]*$'; then
```

---

If the goal is only to avoid `#` (thinking about `'` `<` `>`), how about:
```suggestion
        if commit_titles | grep -E -v '^([-_.a-z]+[,:] )*[-_.a-z]+: [a-z][^#]*$'; then
```

---

This allows unwanted accents.

---

Fine by me, yet I think we should additionally allow `'`.